extensions/interactions/MusicNotesInput/directives/oppia-interactive-music-notes-input.directive.ts

```diff
@@ -217,13 +216,9 @@ angular.module('oppia').directive('oppiaInteractiveMusicNotesInput', [
           scope.VERTICAL_GRID_SPACING = scope.CONTAINER_HEIGHT /
             verticalGridKeys.length;
 
-          var staffTop = computeStaffTop();
-          var staffBottom = computeStaffBottom();
-
           // The farthest edge of the staff. If a note is placed beyond this
           // position, it will be discarded.
-          var RIGHT_EDGE_OF_STAFF_POSITION =
-            element.find('.oppia-music-input-valid-note-area').width();
+          element.find('.oppia-music-input-valid-note-area').width();
```

Comment:
Does this line do anything without being assigned to any variable? If not, remove it.

Reply:
Removed the line.

Reply:
Lgtm!